packages/mpc-tls/prover/src/handlers/proxy.rs

```diff
@@ -0,0 +1,75 @@
+use crate::{
+    proof::notarize::notarize_request,
+    proxy::{Header, ProxyRequest, BLACKLISTED_HEADERS},
+};
+use actix_web::{route, web, HttpRequest, HttpResponseBuilder, Responder};
+use hyper::body;
+use tracing::debug;
+use url::Url;
+
+#[route(
+    "/proxy",
+    method = "GET",
+    method = "POST",
+    method = "PUT",
+    method = "PATCH"
+)]
+pub async fn handle_notarization_request(payload: web::Payload, req: HttpRequest) -> impl Responder {
+    let t_proxy_url = req
+        .headers()
+        .get("T-PROXY-URL")
+        .expect("incomplete headers provided")
+        .to_str()
+        .unwrap();
+
+    let _t_store = req
```

Comment:
`T-STORE` and `T-PUBLISH` will integrate the inter-process messaging to either just store in DB, or also publish to stream. 

Ideally, access to the DB should yield all proofs, and their associated notary public key (ie. a reference to the Notary Public Key via the Notary Gateway).

Consider the Sink Process - a separate process with access to the Node's DB for fetching all attestations and proofs?

Reply:
yes, i already had an elaborate discussion with rafael about this. it is going to be the next thing i tackle now that i am nearly done with the redaction logic.